Resources/Prototypes/Entities/Mobs/Species/slime.yml

```diff
@@ -79,6 +79,16 @@
     - BleedersBite
   - type: ReagentDamageModifier
     modifierSet: SlimeReagents
+  - type: Temperature
+    heatDamageThreshold: 422
```

Comment:
This is significantly past the point of boiling, I do not think Slimes would be happy here.

Reply:
warm :)